### PR TITLE
Add methods to get elevation tiles covered by Valhalla tiles

### PR DESCRIPTION
**Pull request by dnesbitt61**
Add an application (_valhalla_tiles_for_elevation_) to get a list of elevation tiles that are covered within the set of Valhalla tiles specified in the config file. 

Add a script (_valhalla_get_elevation_) to get elevation data using the list of tiles specified in the input file. This list is the output of _valhalla_tiles_for_elevation_. The list could also be constructed using some other script or could be hand-edited if desired. 

The format of the output file from _valhalla_tiles_for_elevation_ and the input file to _valhalla_get_elevation_  is simply each elevation base filename (without the .hgt suffix). Examples are:

N39W079
S25E133

Note there are 2 digits for latitude and 3 digits for longitude (both 0-padded).

### CODE REVIEW

File: scripts/valhalla_get_elevation [hunk 1]
```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+set -e
```
Comment (kevinkreiser) [inline scripts/valhalla_get_elevation]: now we have two elevation scraping scripts. shouldnt we have added the functionality to the python script and kept it to just one elevation script?

Comment (dnesbitt61) [reply scripts/valhalla_get_elevation]: which Python script? My bash skills are insufficient to handle multiple command line argument streams and switching between the 2 somewhat distinct methods (list of elevation files vs. bounding box).

Comment (kevinkreiser) [reply scripts/valhalla_get_elevation]: https://github.com/valhalla/valhalla/blob/master/scripts/valhalla_build_elevation

Comment (kevinkreiser) [reply scripts/valhalla_get_elevation]: no big deal we can reconcile them later

**Pull request merged**